Dockerfile

```diff
@@ -24,9 +26,9 @@ ADD . /cryptpad
 
 WORKDIR /cryptpad
 
-RUN npm install \
-   && npm install -g bower \
-   && bower install --allow-root
+RUN npm install
+RUN npm install -g bower
```

Comment:
If any one of these instructions fail, then CryptPad won't work. Conjoining these commands with `&&` means it will bail out if anything fails.